UM/Qt/Bindings/MainWindow.py

```diff
@@ -44,7 +44,12 @@ def __init__(self, parent = None):
         self.setHeight(int(self._preferences.getValue("general/window_height")))
         self.setPosition(int(self._preferences.getValue("general/window_left")), int(self._preferences.getValue("general/window_top")))
         # Make sure restored geometry is not outside the currently available screens
-        if not self.geometry().intersects(self.screen().availableGeometry()):
+        screenFound = False
```

Comment:
Minor nitpicking, but our coding style uses no caps with underscores for local variables, so you should use screen_found and similar.